avocado/utils/cpu.py

```diff
@@ -138,3 +154,25 @@ def online_cpus_count():
     Return Number of Online cpus in the system
     """
     return os.sysconf('SC_NPROCESSORS_ONLN')
+
+
+def online(cpu):
+    """
+    Online given CPU
+    """
+    process.system('echo 1 > /sys/devices/system/cpu/cpu%d/online' %
+                   int(cpu), shell=True)
```

Comment:
Would you please use python libs to write to file (`open(...).write(...)`) instead of `shell` command.